app/main.py

```diff
@@ -1,13 +1,12 @@
-# from models import Actor
-# from managers import ActorManager
-#
-# if __name__ == "__main__":
-#     Actor.objects = ActorManager()
-#
-#     Actor.objects.create(first_name="Emma", last_name="Watson")
-#     Actor.objects.create(first_name="Daniel", last_name="Radclife")
-#     print(Actor.objects.all())
-#     Actor.objects.update(2, "Daniel", "Radcliffe")
-#     print(Actor.objects.all())
-#     Actor.objects.delete(1)
-#     print(Actor.objects.all())
+from models import Actor
+from managers import ActorManager
+
+if __name__ == "__main__":
+    Actor.objects = ActorManager()
```

Comment:
The task here is to test methods that you created. Simply printing output does not imply a valid test. You can write simple test cases: for example you create Actor with name = "Emma Watson" and on next line get Actor with this name from database to check that record was created. Do the same for all 4 methods